src/main/java/com/laserfiche/repository/api/clients/impl/model/FuzzyType.java

```diff
@@ -5,18 +5,18 @@
 
 public enum FuzzyType {
 
-    NUMBER_0(0),
-    NUMBER_1(1),
-    NUMBER_2(2);
+    NONE("None"),
+    PERCENTAGE("Percentage"),
+    NUMBEROFLETTERS("NumberOfLetters");
```

Comment:
Isn't it better to have this renamed to NUMBER_OF_LETTERS to make it more readable?

Reply:
I updated the code. Thanks!